src/main/java/com/ss/ita/google/ui/pages/SearchResultPage.java

```diff
@@ -54,12 +55,13 @@ public String getResultLinkUrl(int numberResultLink) {
         return getSearchedResultLink(numberResultLink).getAttribute("href");
     }
 
-    public SearchResultPage openResultPage(int pageNumber) {
+    public SearchResultPage openSearchResultPageNumber(int pageNumber) {
```

Comment:
dont use Number in name . U have openSearchResultPage with variable pageNumber. Its obvious.